registries/docker-daemon/start.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Enable local registry
+echo "Installing a local registry"
+docker run -d -p 5000:5000 -p 80:5000 registry:2
```

Comment:
Do we need both port defined? If we switch the k8s service and endpoint to use 80, can we drop 5000?

Reply:
I vaguely remember seeing a problem listening on port 80. Will try again.